modules/administration-guide/pages/caching-images-for-faster-workspace-start.adoc

```diff
@@ -18,52 +18,52 @@ The {image-puller-name-short} provides the following parameters for configuratio
 |===
 |Parameter |Usage |Default
 
-|`CACHING_INTERVAL_HOURS` 
+|`CACHING_INTERVAL_HOURS`
 
 |DaemonSets health checks interval in hours
 |`"1"`
 
-|`CACHING_MEMORY_REQUEST` 
+|`CACHING_MEMORY_REQUEST`
 |The memory request for each cached image when the puller is running. See xref:defining-the-memory-parameters-for-the-image-puller.adoc[].
 |`10Mi`
 
-|`CACHING_MEMORY_LIMIT` 
+|`CACHING_MEMORY_LIMIT`
 |The memory limit for each cached image when the puller is running. See xref:defining-the-memory-parameters-for-the-image-puller.adoc[].
 |`20Mi`
 
-|`CACHING_CPU_REQUEST` 
-|The processor request for each cached image when the puller is running 
+|`CACHING_CPU_REQUEST`
+|The processor request for each cached image when the puller is running
```

Comment:
When you use the present progressive form, in this case `is running`, consider replacing `when` with `while`. Multiple instances.